src/main/java/com/minecolonies/coremod/commands/ActionMenuState.java

```diff
@@ -215,4 +215,61 @@ public int getIntValueForArgument(@NotNull final String argumentName, final int
             return defaultValue;
         }
     }
+
+    @Override
+    public int hashCode()
+    {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((actionMenu == null) ? 0 : actionMenu.hashCode());
+        result = prime * result + ((argumentStateByActionArgumentNameMap == null) ? 0 : argumentStateByActionArgumentNameMap.hashCode());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2583)